### PR TITLE
Speaker fragment, drawer ui improvements

### PR DESCRIPTION
**Pull request by mananwason**
@creativepsyco Please review and merge. I  am not squashing these commits as they are different features.

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 1]
```diff
@@ -104,17 +105,21 @@ private void setUpNavDrawer() {
...
-                public void onClick(View v) {
-                    mDrawerLayout.openDrawer(GravityCompat.START);
-                }
-            });
+            mDrawerLayout.setDrawerListener(mActionBarDrawerToggle);
+            ab.setHomeAsUpIndicator(R.drawable.ic_menu);
+            ab.setDisplayHomeAsUpEnabled(true);
+            ab.setDisplayHomeAsUpEnabled(true);
+            mActionBarDrawerToggle.syncState();
+//            mToolbar.setNavigationIcon(R.drawable.ic_menu);
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Why is this code commented?

File: app/src/main/java/org/fossasia/openevent/data/Speaker.java [hunk 2]
```diff
@@ -160,8 +161,9 @@ public void setId(int id) {
     }
 
     public String generateSql() {
-        String query_normal = "INSERT INTO %s VALUES ('%d', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s','%s');";
-        String query = String.format(query_normal, DbContract.Speakers.TABLE_NAME, id, name, photo, bio, email, web, facebook, twitter, github, linkedin, organisation, position, country);
+        SqlEscapeString escapeString = new SqlEscapeString();
+        String query_normal = "INSERT INTO %s VALUES ('%d', %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s,%s);";
+        String query = String.format(query_normal, DbContract.Speakers.TABLE_NAME, id, escapeString.sqlString(name), escapeString.sqlString(photo), escapeString.sqlString(bio), escapeString.sqlString(email), escapeString.sqlString(web), escapeString.sqlString(facebook), escapeString.sqlString(twitter), escapeString.sqlString(github), escapeString.sqlString(linkedin), escapeString.sqlString(organisation), escapeString.sqlString(position), escapeString.sqlString(country));
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/data/Speaker.java]: Break this line to improve readability

File: app/src/main/java/org/fossasia/openevent/data/Track.java [hunk 3]
```diff
@@ -33,14 +33,11 @@ public String getDescription() {
 
 
     public String generateSql() {
-        String query_normal = "INSERT INTO %s VALUES ('%d', '%s', %s);";
-        String query = String.format(query_normal, DbContract.Tracks.TABLE_NAME, id, name, DatabaseUtils.sqlEscapeString(description));
+        String query_normal = "INSERT INTO %s VALUES ('%d', %s, %s);";
+        String query = String.format(query_normal, DbContract.Tracks.TABLE_NAME, id, DatabaseUtils.sqlEscapeString(name), DatabaseUtils.sqlEscapeString(description));
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/data/Track.java]: Same here

**Comment by mananwason** [comment]
Done @creativepsyco. Please check.

**Comment by creativepsyco** [comment]
Tests are failing and the naming convention is not followed, fixing is required.

**Comment by mananwason** [comment]
@creativepsyco Please check now.

File: app/src/androidTest/java/org/fossasia/openevent/TracksInsertTest.java [hunk 4]
```diff
@@ -29,9 +33,26 @@ private void insertValuesTest() {
         contentValues.put(DbContract.Tracks.NAME, name);
         contentValues.put(DbContract.Tracks.DESCRIPTION, description);
 
-        tracksAssignId = db.insert(DbContract.Tracks.TABLE_NAME, null, contentValues);
+        tracksAssignId = database.insert(DbContract.Tracks.TABLE_NAME, null, contentValues);
         assertTrue(tracksAssignId != -1);
+
     }
+//
+//    public void testinsertValues() {
```
Comment (creativepsyco) [inline app/src/androidTest/java/org/fossasia/openevent/TracksInsertTest.java]: Commented?

**Comment by mananwason** [comment]
@creativepsyco Commented code removed.

**Pull request merged**